### DISCUSSION TITLE
Update Umbrel Release?

### DISCUSSION THREAD

**Discussion by 0ceanSlim** [immich-app/immich#3631] [Q&A] [answered]
I'm running umm immich on Umbrel but the Umbrel Release is behind the current version. Can this be maintained by someone? If someone can show me I can maintain that release. I'm tracking this repo on obtanium already.

**Comment by ddshd** [comment, marked as answer]
Someone was nice enough to submit a PR: https://github.com/getumbrel/umbrel-apps/pull/722
